documentation/rosetta-modelling-component.md

```diff
@@ -1618,26 +1634,24 @@ Conditional mappings allow to build more complex mappings. Conditional mappings
 A set to mapping is used to set the value of an attribute to a constant value. They don\'t attempt to use any data from the input document as the value for the attribute and a synonym value must not be given. The type of the constant must be convertible to the type of the attribute. The constant value can be given as a string (converted as necessary) or an enum - e.g. :
 
 ```
-period PeriodEnum (1..1)
-  [synonym ISDA_Create_1_0 set to PeriodEnum.D]
-itemName string (1..1) <"In this ....">;
-  [synonym DTCC_11_0 set to "comment"]
+engineEnum EngineEnum (0..1)
+  [synonym CONDITIONAL_SET_TO_EXAMPLE_13 set to EngineEnum->HYBRID]
 ```
 
 A set to mapping can be conditional on a [when clause](#when-clause) - e.g.:
 
 ```
-itemName string (1..1) <"In this ....">;
-  [synonym DTCC_11_0 set to "comment" when path = "PartyWorkflowFields.comment"]
+engineEnum EngineEnum (0..1)
+  [synonym CONDITIONAL_SET_TO_EXAMPLE_13 set to EngineEnum->HYBRID when "alternativeFuelDetail" exists and "fuelDetail" exists]
 ```
 
 Multiple set to mappings can be combined in one synonym. They will be evaluated in the order specified with the first matching value used - e.g. :
 
 ```
-xField string (1..1);
-  [synonym Bank_A
-    set to "FISH2" when "b.c.d" = "FISH",
-    set to "SAUSAGE2" when "b.c.d" = "SAUSAGE",
+engineSystem string (1..1)
```

Comment:
no capital letters for combustion, electric, default

Reply:
okey will change that as you said earlier e.g: "HYBRID" to "Hybrid" or "COMBUSTION" to "Combustion" etc

Reply:
done with the new commit